dbetka/harcmap
# Issue/har 14/not making autoupdate calls when network is not available

**Description:**
I've use network capacitor plugin to detect network status changes and show clear communicates about it.
I've use mobile app state to detect if app is active or not and block unnecessary requests.
If app will change its status to active make auto-update.
If auto-update failed cause internet disconnection wait for internet and make auto-update.

**Test cases:**
1. Run app, turn on and off internet connection to see communicates.
2. Run app, turn off internet and wait for information about failed auto-update process, turn on internet and see communicate.

**APK file:** https://drive.google.com/file/d/1i9xWqcSks1JZ0pcdmrEwCzmwQ3OEDLlR/view?usp=sharing
